README.md

```diff
@@ -65,6 +65,19 @@ Model Context Protocol is an emerging standard for enabling AI models and agents
 
 ### Installation
 
+#### Nix
+```bash
+# 1. Clone the repository
+git clone https://github.com/juspay/juspay-mcp.git
+cd juspay-mcp
+
+# 2. Installing dependencies and setting up the environment
+nix develop
```

Comment:
You should add an `.envrc` file containing `use flake`, so people can also use direnv.

Reply:
Done.